src/main/java/com/exasol/validation/GitHubPlatformValidator.java

```diff
@@ -71,10 +69,20 @@ protected void validateContainsHeader(final ReleaseLetter changes) {
     protected void validateGitHubTickets(final ReleaseLetter changesFile) {
         final List<String> wrongTickets = collectWrongTickets(changesFile);
         if (!wrongTickets.isEmpty()) {
+            reportWrongTickets(changesFile.getFileName(), wrongTickets);
+        }
+    }
+
+    private void reportWrongTickets(final String fileName, final List<String> wrongTickets) {
+        final String wrongTicketsString = String.join(", ", wrongTickets);
+        if (this.branchContent.isDefaultBranch()) {
             throw new IllegalStateException(
                     "E-RR-VAL-2: Some of the mentioned GitHub issues are not closed or do not exists: "
-                            + String.join(", ", wrongTickets) + ", Please, check the issues numbers in your '"
-                            + changesFile.getFileName() + "' one more time.");
+                            + wrongTicketsString + ", Please, check the issues numbers in your '" + fileName
+                            + "' one more time.");
+        } else {
+            LOGGER.info("Don't forget to close the tickets mentioned in the '" + fileName
```

Comment:
in the ticket description a *warning* is requested. Maybe use `LOGGER.warn` instead?

Reply:
Good idea!